apps/api/src/controllers/form.controller.ts

```diff
@@ -111,6 +113,32 @@ export class FormController {
 		return await this.form.createFormField(payload, formId);
 	}
 
+	@Patch('/form/:orgId/schema/:formId/:fieldId')
+	@Roles(Role.ADMIN, Role.EDITOR)
```

Comment:
We should move from roles tag here. Makes the code more verbose

Reply:
how do you want me to do it ?

Reply:
ignore. We can work on it later